content/developer/howtos/rdtraining/B_acl_irrules.rst

```diff
@@ -88,27 +88,26 @@ the ``res.groups`` model. They are normally part of a module's :ref:`master data
 
 As simple example `can be found here <https://github.com/odoo/odoo/blob/532c083cbbe0ee6e7a940e2bdc9c677bd56b62fa/addons/hr/security/hr_security.xml#L9-L14>`_.
 
-.. admonition:: what is the *category_id*?
-
-    ``category_id`` a *module category*, it is automatically generated from the
-    :ref:`category defined in module manifest files <howto/rdtraining/03_newapp>`.
+.. exercise::
 
-    .. todo:: the demo module needs to have the category set, to
-              *Real Estate/Brokerage*, ideally the newapp part would already do
-              that
+    * Create the ``security.xml`` file in the appropriate folder and add it to the ``__manifest__.py`` file.
 
-.. exercise::
+    * If not already, add a ``'category'`` field to your ``__manifest__.py`` with value ``Real Estate/Brokerage``.
 
-    Create the ``security.xml`` file in the appropriate folder and add it to the
-    ``__manifest__.py`` file.
+    * Add a record creating a group with the id ``estate_group_user``, the name "Agent"
+      and the category ``base.module_category_real_estate_brokerage``.
 
-    Add a record creating a group with the id ``estate_group_user``, the name
-    "Agent" and the category ``base.module_category_real_estate_brokerage``.
+      Note: Where does that category comes from ?
+        It's a *module category*.
+        Here we used the category id ``base.module_category_real_estate_brokerage``
+        which was automatically generated by Odoo based on the `category` set in the ``__manifest__.py`` of the module.
+        You can also find here the list of
+        `default module categories <https://github.com/odoo/odoo/blob/71da80deb044852a2af6b111d695f94aad7803ac/odoo/addons/base/data/ir_module_category_data.xml>`_
+        provided by Odoo.
```

Comment:
Use the [note](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#note) alert block to insert notes.
```suggestion
      .. note::
         Where does that category comes from? It's a *module category*. Here we used the category id
         `base.module_category_real_estate_brokerage` which was automatically generated by Odoo
         based on the `category` set in the `__manifest__.py` of the module. You can also find here
         the list of `default module categories
         <https://github.com/odoo/odoo/blob/71da80deb044852a2af6b111d695f94aad7803ac/odoo/addons/base/data/ir_module_category_data.xml>`_
         provided by Odoo.
```